### ISSUE TITLE
Need support for previous editions of the professional C++ book

### ISSUE THREAD

**Issue by mguarin0** [Professional-CPP/edition-6#36]
It would be very helpful to be able to access code for previous versions of the professional c++ book

**Comment by MichalRK19780524** [comment]
Unfortunately, it is not possible to access the source code for the C++ professional 5th edition book from the publisher's website

**Comment by MarcGregoire** [comment]
You're right. It seems the download links disappeared for the 5th edition. They are still there for the 4th edition.
I have contacted the publisher and asked them to add the links back for the 5th edition as well.

**Comment by mguarin0** [comment]
Thank you!

**Comment by MarcGregoire** [comment]
@mguarin0 The links have been added back for the 5th edition as well.

**Comment by mguarin0** [comment]
sorry for late reply! thank you so much!

**Comment by mguarin0** [comment]
thank you!
